### PR TITLE
Test bundle switch

### PR DESCRIPTION
**Pull request by Michkov**

### CODE REVIEW

File: .tekton/cache-pull-request.yaml [hunk 1]
```diff
@@ -20,8 +20,11 @@ spec:
       value: cache/src/main/docker/Dockerfile.all-in-one
   pipelineRef:
     name: docker-build
-    bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
+    bundle: quay.io/redhat-appstudio/build-templates-bundle:latest
```
Comment (gabemontero) [inline .tekton/cache-pull-request.yaml:23]: they got rid of `latest` tag @Michkov 

it is now `devel`

see https://github.com/redhat-appstudio/jvm-build-service/pull/302

Comment (gabemontero) [reply .tekton/cache-pull-request.yaml:23]: I've reopened 302 for comparison

**Comment by stuartwdouglas** [comment]
Looks like this worked?

**Comment by gabemontero** [comment]
> Looks like this worked?

it did @stuartwdouglas .... we ended up using https://github.com/redhat-appstudio/jvm-build-service/pull/302 for this

**Pull request closed without merging**